app/handlers/ErrorHandler.scala

```diff
@@ -26,17 +26,18 @@ import views.html.{ErrorTemplate, not_found}
 import javax.inject.{Inject, Singleton}
 
 @Singleton
-class ErrorHandler @Inject()(
-                              val messagesApi: MessagesApi,
-                              view: ErrorTemplate,
-                              notFound: not_found,
-                            )(implicit appConfig: FrontendAppConfig) extends FrontendErrorHandler with I18nSupport {
+class ErrorHandler @Inject()(val messagesApi: MessagesApi,
+                             view: ErrorTemplate,
+                             notFound: not_found
+                            )(implicit appConfig: FrontendAppConfig)
+  extends FrontendErrorHandler with I18nSupport {
 
-  override def standardErrorTemplate(pageTitle: String, heading: String, message: String)(implicit rh: Request[_]): Html =
-    view(pageTitle, heading, message)
+  override def standardErrorTemplate(pageTitle: String,
```

Comment:
It can be reformat like below

  override def standardErrorTemplate(pageTitle: String,
                                     heading: String,
                                     message: String)(implicit rh: Request[_]): Html =
    view(pageTitle, heading, message)